### PR TITLE
V0.16e

### PR DESCRIPTION
**Pull request by jblance**

### CODE REVIEW

File: powermon/commands/command.py [hunk 1]
```diff
@@ -23,26 +23,35 @@ def __str__(self):
         for output in self.outputs:
             _outs += str(output)
 
-        return f"Command: {self.name}, type: {self.type}, outputs: [{_outs}] last run: {last_run}, next run: {next_run}, {self.trigger}, defn: {self.command_defn}"
+        return f"Command: {self.name=} {self.full_command=}, {self.type=}, [{_outs=}], {last_run=}, {next_run=}, {str(self.trigger)}, {self.command_defn=}"
 
-    def __init__(self, config):
+    @classmethod
```
Comment (rossandrews) [inline powermon/commands/command.py:28]: Looks good!

File: powermon/dto/commandDTO.py [hunk 2]
```diff
@@ -1,6 +1,7 @@
 from pydantic import BaseModel
 
 
+# TODO: update
```
Comment (rossandrews) [inline powermon/dto/commandDTO.py:4]: What would you like updated on the DTOs?

**Review by rossandrews — COMMENTED** [review]
Sorry for disappearing for a bit, went on a vacation and took some time to get back into my routine.

Lot of great changes in here, looking forward to working with them.

**Review by rossandrews — APPROVED** [review]

File: makefile
Comment (rossandrews) [inline makefile:1]: I was just reading up docker-compose standalone vs docker compose plugin. Looks like you are ahead of me.

File: powermon/dto/commandDTO.py (hunk 2 above)
Comment (jblance) [reply powermon/dto/commandDTO.py:4]: its just a reminder that they need to be updated to cater for the changes that are being made

**Pull request merged**